### DISCUSSION TITLE
Quote template : Extract summary values

### DISCUSSION THREAD

**Discussion by Ilamparithi-Tamizhanangu** [YetiForceCompany/YetiForceCRM#15742] [How to] [answered]
Hi,

I am using below template to generate the product table in the quotation module.

template : 
$(inventory : type=table columns=seq,comment1,qty,total,discount,net,tax , href=no , labels=SNo,Product Description,QTY, Product Price,Discount,Price after discount,Tax , width=10px,250px,10px,70px,70px,70px,60px)$

output : 
![image](https://user-images.githubusercontent.com/96602176/155279113-7dbb6d72-bd30-4129-9cf9-8dda498415f2.png)

The last row display the sum of the values in the particular column. 

my queries:
1. How to remove the last row displayed in the price table.
2. is it possible to get the fieldnames to display it in the different format?

I wish to display the product price & summary separately.  
![image](https://user-images.githubusercontent.com/96602176/155279782-cac48fae-6484-4740-8e6a-178ce6ce6b73.png)

Thanks & regards,
Rajasekaran.

**Comment by Ilamparithi-Tamizhanangu** [comment]
hi,

I calculated the price summary separately. Could you please share the procedure to remove the last row (sum of the column values) in the table. 

Thanks & regards,
Rajasekaran.

**Reply by vovpff** [reply]
You need to write textparsef function what render products table

**Comment by Ilamparithi-Tamizhanangu** [comment, marked as answer]
hi,

Thanks for the update. Now modified the textpharser file. 

Thanks & regards,
Rajasekaran
